service/chain/disk.go

```diff
@@ -185,6 +185,42 @@ func (d *Disk) Results(height uint64) ([]*flow.TransactionResult, error) {
 	return converted, nil
 }
 
+func (d *Disk) Seals(height uint64) ([]*flow.Seal, error) {
+
+	blockID, err := d.block(height)
+	if err != nil {
+		return nil, fmt.Errorf("could not get block for height: %w", err)
```

Comment:
WDYT?

```suggestion
		return nil, fmt.Errorf("could not get block for height %d: %w", height, err)
```

Reply:
Yeah, I like returning all available (relevant) information in the error. However I saw that other functions don't return it, so I maintained suite.. Honestly I'd rather add it everywhere